pkg/build/build.go

```diff
@@ -0,0 +1,17 @@
+package build
```

Comment:
I have plans to split out the build logic into this package in another PR. This is in prep for #361.